README.md

```diff
@@ -23,7 +23,7 @@ A browser interface based on Gradio library for Stable Diffusion.
     - use multiple embeddings with different numbers of vectors per token
     - works with half precision floating point numbers
     - train embeddings on 8GB (also reports of 6GB working)
-- Extras tab with:
+- Extra tabs with:

```

Comment:
![image](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/40751091/74f1f341-c587-4ecb-aa8c-bc7e2e574663)

Reply:
i suppose it was my mistake then,,,,